packages/fast-components-react-base/src/toggle/toggle.props.ts

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+
+export interface IToggleProps {
+    children?: React.ReactNode | React.ReactNode[];
```

Comment:
Is this really a place where we want to support anything?

Reply:
In the label... sure why not? I don't think it needs to be text-only. If they want a glyph or hyperlink, they should be able to add one